sygnal/sygnal.py

```diff
@@ -89,26 +85,25 @@ def __init__(self, config, custom_reactor, tracer=opentracing.tracer):
                 )
                 config["proxy"] = proxy_url
 
-        # Old format db config
-        if config.get("db") is not None:
-            logger.warning(
-                "Config includes the legacy 'db' option, please migrate"
-                " to 'database' instead"
-            )
-            config["database"] = {
-                "name": "sqlite3",
-                "args": {"dbfile": config["db"]["dbfile"]},
-            }
-        elif config.get("database") is None:
-            config["database"] = {"name": "sqlite3", "args": {"dbfile": "sygnal.db"}}
-
         sentrycfg = config["metrics"]["sentry"]
         if sentrycfg["enabled"] is True:
             import sentry_sdk
 
             logger.info("Initialising Sentry")
             sentry_sdk.init(sentrycfg["dsn"])
 
+        if config.get("db") is not None:
+            logger.warning(
+                "Config includes the legacy 'db' option and will be ignored"
+                "as Sygnal no longer uses a DB, this field can be removed"
```

Comment:
```suggestion
                " as Sygnal no longer uses a DB, this field can be removed"
```

(not obvious but the two strings will be joined back-to-back, so need another space here)